de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/InstantIncomingProjectNegotiation.java

```diff
@@ -39,7 +43,8 @@
     private static final Logger log = Logger
         .getLogger(InstantIncomingProjectNegotiation.class);
 
-    private final ProjectActivityQueuer activityQueuer = new ProjectActivityQueuer();
+    private ResourceActivityQueuer activityQueuer;
+    private StartHandle stoppedLocalEditor;
```

Comment:
Ok, then the confusion stems from me working with IntelliJ. There, an "editor" is a window/tab in the main project view that can be used to modify a specific file. Each editor belongs to exactly one file. There can be any number of editors open at once.

Reading it from this perspective is a bit confusing as you are not blocking a specific editor but the entire IDE/any edits. This is why I would prefer to avoid the term "editor".